python/ray/constants.py

```diff
@@ -0,0 +1,4 @@
+# OS names:
+WINDOWS_OS = "Windows"
```

Comment:
we already have `python/ray/_private/ray_constants.py`.

For this case, I actually think it's fine to use the string literals directly. These string literals will not change.

Reply:
I agree, thanks.